scripts/add_attribute.py

```diff
@@ -138,4 +138,4 @@
 
 with open(fileName, "wb") as f:
     for line in newLines:
-        f.write(line + "\n")
+        f.write((line + "\n").encode())
```

Comment:
This should be a separate PR, but this looks like confusing str and bytes before writing.

Reply:
I'll move this to a different PR, I think the real problem is that the file is opened in binary mode (`wb`)